pkg/osd_cluster_upgrader/config.go

```diff
@@ -4,19 +4,18 @@ import (
 	"fmt"
 	"time"
 
-	"github.com/openshift/managed-upgrade-operator/pkg/machinery"
+	"github.com/openshift/managed-upgrade-operator/pkg/drain"
 )
 
 type osdUpgradeConfig struct {
-	Maintenance maintenanceConfig   `yaml:"maintenance"`
-	Scale       scaleConfig         `yaml:"scale"`
-	NodeDrain   machinery.NodeDrain `yaml:"nodeDrain"`
-	HealthCheck healthCheck         `yaml:"healthCheck"`
+	Maintenance maintenanceConfig `yaml:"maintenance"`
+	Scale       scaleConfig       `yaml:"scale"`
+	NodeDrain   drain.NodeDrain   `yaml:"nodeDrain"`
+	HealthCheck healthCheck       `yaml:"healthCheck"`
 }
 
 type maintenanceConfig struct {
 	ControlPlaneTime int           `yaml:"controlPlaneTime"`
-	WorkerNodeTime   int           `yaml:"workerNodeTime"`
```

Comment:
Created PR https://github.com/openshift/osde2e/pull/583 for osde2e to reflect these changes in the configmap. More changes can be added to it.